src/packages/engine-core/src/NodeEngine.ts

```diff
@@ -1082,8 +1075,7 @@ ${this.lastErrorLog.fields.file}:${this.lastErrorLog.fields.line}:${this.lastErr
         const isError = await this.handleRequestError(e, numTry < 3)
         if (!isError) {
           // retry
-          if (numTry < 3) {
-            await new Promise((r) => setTimeout(r, Math.random() * 1000))
```

Comment:
No more random waiting time?

Reply:
Correct. It turned out to be unnecessary and one source for the flaky behavior.